### ISSUE TITLE
hello world

### ISSUE THREAD

**Issue by NiklasTR** [labdao/gui-wrapper#1]
Hi @jlevers I created a repo for the GUI wrapping work that we have been talking about. Would you be interested in being the champion here? 

Some services I believe we could wrap: 
* swissADME - http://www.swissadme.ch/index.php
* naturalantibody - https://research.naturalantibody.com/

**Comment by jlevers** [comment]
yep definitely! i think once i figure out how to use ceramic effectively we'll be able to start moving fast on this

**Comment by blahah** [comment]
I have a working example using vercel's free edge workers. Should be a good basis for getting up and running quickly, let me push it up to github.

**Comment by blahah** [comment]
You should be able to use [chrome devtools in chrome/chromium/brave](https://developer.chrome.com/docs/devtools/recorder/) to record your interactions on the webapp you want to wrap, and then export the sequence of interactions and have puppeteer run in by pasting them into the example in the previous comment.
